app/models/video.rb

```diff
@@ -1,5 +1,6 @@
 class Video < ActiveRecord::Base
   belongs_to :category
+  has_many :reviews #, order: "created_at DESC"
```

Comment:
Rails 4+ uses a different syntax. You have to provide a lambda.

```ruby
has_many :reviews, -> { order "created_at DESC" }
```

Order is now a method which accepts a string or a hash `order("created_at DESC")` or `order(created_at: :desc)`